frontend/src/metabase/dashboard/hoc/DashboardControls.tsx

```diff
@@ -61,6 +65,14 @@ export const DashboardControls = <T extends DashboardControlsProps>(
 
     useSyncURLSlug({ location });
 
+    useEffect(() => {
```

Comment:
The embed frame in `StaticEmbed` seems to rely on the `font` being set from the store (or at least that's the impression I got from solving tests)

Reply:
I was confused at first since it looks a lot like this.
https://github.com/metabase/metabase/blob/911e76ba524cc281c5d469ec70a035ac5b6becaa/enterprise/frontend/src/embedding-sdk/components/public/MetabaseProvider.tsx#L36-L42

Does this change affect the SDK, or does it change the behavior for public and static embeddings too?

Reply:
It affects the static embeddings at least. The test that was failing was one where the font wasn't being set within the static embed modal when customizing the dashboard for publishing

As for the `does it affect the SDK` - it shouldn't. `DashboardControls` shouldn't be used in the SDK context since it handles the `location` values